home/tests/test_assessment_import_export.py

```diff
@@ -618,6 +655,64 @@ def test_invalid_medium_score(self, csv_impexp: ImportExport) -> None:
         )
         assert e.value.row_num == 2
 
+    def test_multiple_missing_headers(self, csv_impexp: ImportExport) -> None:
+        """
+        Importing a CSV with multiple missing headers should return an error
+        stating all the mandatory headers that are missing
+        """
+        with pytest.raises(ImportAssessmentException) as e:
+            csv_impexp.import_file("assessments_multiple_missing_headers.csv")
+        assert (
+            e.value.message == "Missing mandatory headers: title, slug, "
+            "generic_error, locale"
+        )
+        assert e.value.row_num == 1
+
+    def test_missing_title(self, csv_impexp: ImportExport) -> None:
+        """
+        Importing a CSV with a missing title field should return an error
+        that a title is missing
+        """
+        with pytest.raises(ImportAssessmentException) as e:
+            csv_impexp.import_file("assessments_missing_title.csv")
+        assert e.value.message == "The import file is missing required fields: title"
+        assert e.value.row_num == 4
+
+    def test_missing_locale(self, csv_impexp: ImportExport) -> None:
+        """
+        Importing a CSV with a missing locale field should return an error
+        that a locale is mmissing
+        """
+        with pytest.raises(ImportAssessmentException) as e:
+            csv_impexp.import_file("assessments_missing_locale.csv")
+        assert e.value.message == "The import file is missing required fields: locale"
+        assert e.value.row_num == 5
+
+    def test_missing_generic_error(self, csv_impexp: ImportExport) -> None:
+        """
+        Importing a CSV with a missing generic error field should return an error
+        that a generic error is mmissing
+        """
+        with pytest.raises(ImportAssessmentException) as e:
+            csv_impexp.import_file("assessments_missing_generic_error.csv")
+        assert (
+            e.value.message
+            == "The import file is missing required fields: generic_error"
+        )
+        assert e.value.row_num == 2
```

Comment:
We probably only need one of these, since we test the "all columns missing" case above.